recipes/jwt-cpp/all/conanfile.py

```diff
@@ -27,21 +27,26 @@ def export_sources(self):
     def requirements(self):
         self.requires("openssl/1.1.1q")
         if not self._supports_generic_json:
-            self.requires("picojson/1.3.0")
+            self.requires("picojson/cci.20210117")
```

Comment:
I think it's better to point to official releases when they are available

```suggestion
            self.requires("picojson/cci.1.3.0")
```

Reply:
Fixed